tests/c01 - GendtseWaardNevengeul/Qmin_4000_v2.cfg

```diff
@@ -13,14 +13,14 @@
 [C1]
   Discharge   = 3000.0
   Reference   = reference-Q1_map.nc
-  WithMeasure = reference-Q1_map.nc
+  WithIntervention = reference-Q1_map.nc
 
 [C2]
   Discharge   = 4000.0
   Reference   = reference-Q2_map.nc
-  WithMeasure = reference-Q2_map.nc
+  WithIntervention = reference-Q2_map.nc
 
 [C3]
   Discharge   = 6000.0
   Reference   = reference-Q3_map.nc
-  WithMeasure = measure-Q3_map.nc
+  WithIntervention = measure-Q3_map.nc
```

Comment:
is this the correct file name?

Reply:
lets have a look together

Reply:
@MRVermeulenDeltares also update the related images for the documentation